### PR TITLE
Add ScottoLong Keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
Add ScottoLong handwired keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/scottokeebs/scottolong/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Compiling
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/jscotto/scottolong:default
+
+Flashing example for this keyboard:
+
+    make handwired/jscotto/scottolong:default
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottolong/readme.md:19]:
```suggestion
    make handwired/scottokeebs/scottolong:default

Flashing example for this keyboard:

    make handwired/scottokeebs/scottolong:default
```

File: keyboards/handwired/scottokeebs/scottolong/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# ScottoLong
+
+![ScottoLong](https://i.imgur.com/AEQWXLlh.jpeg)
+
+A 33 ortholinear board with a 7u spacebar. Case files available [here](https://github.com/joe-scotto/scottokeebs).
+
+-   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+-   Hardware Supported: RP2040
+-   Hardware Availability: [Amazon](https://amazon.com)
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottolong/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
* Hardware Supported: RP2040
* Hardware Availability: [Amazon](https://amazon.com)
```

File: keyboards/handwired/scottokeebs/scottolong/info.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            "GP8",
+            "GP9",
+            "GP10",
+            "GP11",
+            "GP12",
+            "GP13",
+            "GP14",
+            "GP15"
+        ],
+
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottolong/info.json:28]:
```suggestion
        "cols": ["GP6", "GP7", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Please revert the submodule changes.

File: keyboards/handwired/scottokeebs/scottolong/info.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    "matrix_pins": {
+        "cols": ["GP6", "GP7", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+        "rows": ["GP2", "GP3", "GP4", "GP5"]
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0011",
+        "vid": "0xFEED"
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottolong/info.json:24]: Vendor ID cannot be FEED for VIA support.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scottolong/info.json (hunk 4 above)
Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scottolong/info.json:24]: Resolved in [f3bffe3](https://github.com/qmk/qmk_firmware/pull/20938/commits/f3bffe37a89f128dd7692878a19ab63f2a6ff649). Board has Vial on my site for GUI support, VIA not needed.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**